pkg/windows/windows.go

```diff
@@ -402,12 +400,6 @@ func (vm *windows) Run(cmd string, psCmd bool) (string, error) {
 
 	out, err := vm.interact.run(cmd)
 	if err != nil {
-		// Hack to not print the error log for "sc.exe qc" returning 1060 for non existent services
-		// and not print error when the command takes too long to return after removing HNS networks.
-		if !(strings.Contains(cmd, serviceQueryCmd) && strings.Contains(out, serviceNotFound)) &&
-			!(strings.Contains(err.Error(), cmdExitNoStatus) && strings.HasSuffix(cmd, removeHNSCommand+";\"")) {
-			vm.log.Error(err, "error running", "cmd", cmd, "out", out)
-		}
```

Comment:
Is there going to be a follow-up PR to re-introduce this log at the call site(s)?